backend/src/logger/api.logger.ts

```diff
@@ -1,4 +1,4 @@
-import * as pine from "pine";
+import pine from "pine";
```

Comment:
This was the error that was failing for the app to run. I got an error in the command saying can't compile the typescript. Will try this and see how it works.

Reply:
@shyakadev Well done. My approach was a bit different but I think this works as well.

Reply:
Looks like Pine did us badly😂😂😂😂😂

Reply:
@marksikaundi did you try

Reply:
@shyakadev yeah tried it